### DISCUSSION TITLE
Move participation spec to TIPs

### DISCUSSION THREAD

**Discussion by Dr-Electron** [iotaledger/tips#124] [Ideas]
I think we should move the [participation spec](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) to this repo. Shouldn't this be an (application) TIP?

**Comment by Phyloiota** [comment]
Absolutely agree. Alex and I discussed this already, but it got buried under all the other work

**Reply by Dr-Electron** [reply]
If people agree I can just start a PR and help out
